GitCommands/Git/GitItemStatus.cs

```diff
@@ -182,6 +182,29 @@ private void SetFlag(bool isSet, Flags flag)
 
         #endregion
 
+        #region Derived Flags
+
+        /// <summary>
+        /// Indicates whether the Git item was added in the (artificial or real) commit.
+        /// </summary>
+        public bool IsAdded
+            => IsNew || IsCopied;
+
+        /// <summary>
+        /// Indicates whether the Git item is yet to be committed;
+        /// that is it belongs to either WorkTree or Index (<see cref="Staged"/> must be set).
+        /// </summary>
+        public bool IsUncommitted
+            => Staged is (StagedStatus.WorkTree or StagedStatus.Index);
+
+        /// <summary>
+        /// Indicates whether the Git item is new or copied and has not been committed yet.
```

Comment:
I did not suggest a summary here (as for the others), as it is hard to explain. 
Adding references at least.
```suggestion
        /// Indicates whether the Git item is new or copied (<see cref="IsAdded"/>) and has not been committed yet (<see cref="IsUncommitted"/>).
```